src/libraries/System.Collections.Immutable/src/System/Collections/Frozen/String/KeyAnalyzer.cs

```diff
@@ -235,15 +221,23 @@ private static bool ContainsAnyLetters(ReadOnlySpan<char> s)
 #endif
         }
 
-        private static double GetUniquenessFactor(HashSet<string> set, ReadOnlySpan<string> uniqueStrings)
+        private static bool HasSufficientUniquenessFactor(HashSet<string> set, ReadOnlySpan<string> uniqueStrings)
         {
             set.Clear();
+
+            // SufficientUniquenessFactor of 95% is good enough.
```

Comment:
This constant was deleted.

Reply:
I was too lazy/tired to turn the const name into three separate words. If you don't mind I am going to do that in next PR.